src/__tests__/core/block.ommers.test.ts

```diff
@@ -0,0 +1,31 @@
+import { testGraphql } from '../utils';
+
+const { execQuery } = testGraphql();
+
+test('block->ommer: successfully returns ommer account', async () => {
+  const query = `
+      {
+        block(number: 5771884) {
+          ommers {
+            hash
+          }
+        }
+      }`;
+
+  const expected = {
+    data: {
+      block: {
+        ommers: [
+          {
+            hash: '0x239a3ffee425204affbd8aca664ebb6c405c0fb2ccb2bbaf93d9e7c965bd03dc',
```

Comment:
Fetching only the hash is insufficient to test that we can correctly fetch nested fields of the Block, as the ommer hashes are provided in the result of the `getBlock()` call. We want to test other fields like the miner or the state root.